apps/openmw/mwmechanics/aicombat.cpp

```diff
@@ -442,10 +442,9 @@ namespace MWMechanics
 
         if (targetClass.hasInventoryStore(target))
         {
-            MWMechanics::WeaponType weapType = WeapType_None;
-            MWWorld::ContainerStoreIterator weaponSlot =
-                MWMechanics::getActiveWeapon(targetClass.getCreatureStats(target), targetClass.getInventoryStore(target), &weapType);
-            if (weapType != WeapType_PickProbe && weapType != WeapType_Spell && weapType != WeapType_None && weapType != WeapType_HandToHand)
+            int weapType = ESM::Weapon::None;
+            MWWorld::ContainerStoreIterator weaponSlot = MWMechanics::getActiveWeapon(target, &weapType);
+            if (weapType != ESM::Weapon::PickProbe && weapType != ESM::Weapon::Spell && weapType != ESM::Weapon::None && weapType != ESM::Weapon::HandToHand)
```

Comment:
I feel like there either could be a Lesser comparison with None type, or an opportunity for another flag for some of these types.